soccer-records-model/src/main/java/cz/muni/fi/pa165/team/TeamPlayer.java

```diff
@@ -81,24 +81,49 @@ public String getFirstname()
         return firstname;
     }
 
+    public void changeFirstname(String firstname)
+    {
+        this.firstname = firstname;
+    }
+
     public String getSurname()
     {
         return surname;
     }
 
+    public void changeSurname(String surname)
+    {
+        this.surname = surname;
+    }
+
     public int getHeight()
     {
         return height;
     }
 
+    public void changeHeight(int height)
+    {
+        this.height = height;
+    }
+
     public int getWeight()
     {
         return weight;
     }
 
+    public void changeWeight(int weight)
+    {
+        this.weight = weight;
```

Comment:
* assert na notNull
* `Assert.isTrue(height > 0, "Weight must be a positive integer");`